main.go

```diff
@@ -138,7 +138,7 @@ func ParseArgs() (string, []string) {
 	// non-zero exit status.
 	if len(os.Args) == 1 {
 		_ = Help(command)
-		os.Exit(1)
+		os.Exit(0)
```

Comment:
exit 1 seems correct here? For example, the command `sda-cli createKey` lacks the required `<name>` and is invalid.